test-suite-ex/js/src/test/scala/org/scalajs/testsuite/jsinterop/GlobalScopeTestEx.scala

```diff
@@ -73,6 +73,28 @@ class GlobalScopeTestEx {
     assertEquals("evil thiz babar", bar.foo())
   }
 
+  @Test def accessGlobalThis(): Unit = {
```

Comment:
I don't think these two tests need to be in the `test-suite-ex`. They don't have weird global effects on the whole codebase.

I think the ones using an ` belong to `compiler.InteroperabilityTest` and `jsinterop.MiscInteropTest`. See for example tests involving `InteroperabilityTestGlobalScope` and `InteroperabilityTestGlobalValsAndDefs` in the former. Tests for `typeOf` in particular would belong in `MiscInteropTest`.

Reply:
Yeah, I moved those tests into `compiler.InteroperabilityTest` and `jsinterop.MiscInteropTest`. (I couldn't make `assumeTrue(noModule)` work correctly in `test-suite-ex` 🤔 )

> weird global effects on the whole codebase

I see, that's the criteria for push tests into the `test-suite-ex`.